retro_data_structures/formats/ancs.py

```diff
@@ -145,34 +150,37 @@ def _yield_dependency_array(asset_ids: Optional[List[int]], asset_type: str, gam
         for asset_id in asset_ids:
             yield from _yield_dependency_if_valid(asset_id, asset_type, game)
```

Comment:
are these functions still used?

Reply:
the legacy dependencies stuff are used by RDV I'm pretty sure. I needed to add them back to get the game to export